Basic/APIDiorama/Assets/Common/Scripts/com.unity.samples.api-diorama.common.asmdef

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "com.unity.samples.api-diorama.common",
+    "rootNamespace": "",
+    "references": [
+        "Unity.Netcode.Runtime",
+        "Unity.Netcode.Components",
+        "Unity.InputSystem",
+        "Unity.Collections",
+        "Unity.TextMeshPro",
+        "DevLocker.Utils.SceneReference"
```

Comment:
Just a note to remove the reference here and in the other namespaces as well.

Reply:
Nice catch, fixed!